### PR TITLE
Add LED manager function to sysi.c to 2100, 2410 and 2700 systems

### PR DESCRIPTION
**Pull request by sholeksandr**
Add LED manager function to sysi.c to 2100, 2410 and 2700 systems
Add API for read min_fan_speed.
Fix led_mode compare ledi.c

### CODE REVIEW

File: packages/platforms/mellanox/x86-64/x86-64-mlnx-msn2100/onlp/builds/src/module/src/fani.c [hunk 1]
```diff
@@ -348,3 +348,14 @@ onlp_fani_ioctl(onlp_oid_t id, va_list vargs)
...
 
+int 
+onlp_fani_get_min_rpm(int id)
+{
+    int   len = 0, nbytes = 10;
+    char  r_data[10]   = {0};
+    char  fullpath[65] = {0};
+
+    snprintf(fullpath, sizeof(fullpath), "%s%s", PREFIX_PATH, fan_path[id].min);
+    OPEN_READ_FILE(fullpath, r_data, nbytes, len);
```
Comment (jnealtowns) [inline packages/platforms/mellanox/x86-64/x86-64-mlnx-msn2100/onlp/builds/src/module/src/fani.c]: Please convert to onlp_file_read() and family.

**Comment by sholeksandr** [comment]
I was changed OPEN_READ_FILE macros to the onlp_file_read() API.

**Comment by jnealtowns** [comment]
@sholeksandr This pull request is now conflicted.

**Comment by jnealtowns** [comment]
Thank you @sholeksandr !

**Pull request merged**